.travis.yml

```diff
@@ -2,6 +2,8 @@ dist: bionic
 language: node_js
 node_js:
   - 12.16.1
+before_install:
+  - npm install -g @lhci/cli@0.8.x
```

Comment:
Can we move it to lhci.sh instead? So it's only installed for one task (not all of them)

Reply:
yep